content/blog/2024/09/2024-09-22.md

```diff
@@ -263,6 +237,36 @@ Matrix SDK written in pure Dart.
 > * Discussion, suggestions and ideas are welcome in #libmatrix:rory.gay (Space: #mru-space:rory.gay)
 > * Got a cool project that you're working on and want to share, using LibMatrix? Be sure to let us know, we'd love to hear all about it!
 
+## Dept of Ops 🛠
```

Comment:
That's not right is it?

Reply:
It is. The diff is just a bit weird. Matrix Utils is an ops tool (afaik?)  and the LibMatrix one was moved one level up to the SDKs

Reply:
Github just shows the wrong thing as being moved here

Reply:
(I seem to have missed the "Emma reported" part though...)

Reply:
Opened https://github.com/matrix-org/matrix.org/pull/2503 for that

Reply:
> Matrix Utils is an ops tool (afaik?)  and the LibMatrix one was moved one level up to the SDKs

TIL!